src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/transpose.cpp

```diff
@@ -1,159 +1,44 @@
 namespace {
 std::map<std::string, std::string> additional_config;
 
+const auto cpuParams_ndhwc = CPUSpecificParams {{ndhwc}, {}, {}, {}};
+const auto cpuParams_ncdhw = CPUSpecificParams {{ncdhw}, {}, {}, {}};
+
 const auto cpuParams_nChw16c = CPUSpecificParams {{nChw16c}, {}, {}, {}};
 const auto cpuParams_nCdhw16c = CPUSpecificParams {{nCdhw16c}, {}, {}, {}};
 
 const auto cpuParams_nChw8c = CPUSpecificParams {{nChw8c}, {}, {}, {}};
 const auto cpuParams_nCdhw8c = CPUSpecificParams {{nCdhw8c}, {}, {}, {}};
 
-const auto cpuParams_nhwc = CPUSpecificParams {{nhwc}, {}, {}, {}};
-const auto cpuParams_ndhwc = CPUSpecificParams {{ndhwc}, {}, {}, {}};
-
-const auto cpuParams_nchw = CPUSpecificParams {{nchw}, {}, {}, {}};
-const auto cpuParams_ncdhw = CPUSpecificParams {{ncdhw}, {}, {}, {}};
-
 const std::vector<InferenceEngine::Precision> netPrecisions = {
-        Precision::I8,
         Precision::BF16,
-        Precision::FP32
-};
-
-const std::vector<InferenceEngine::Precision> netPrecisionsPerChannels = {Precision::I8, Precision::FP32};
-
-const std::vector<InputShape> staticInputShapes4DC16 = {InputShape{// dynamic
-                                                                   {-1, 16, -1, -1},
-                                                                   // target
-                                                                   {{2, 16, 21, 10}, {3, 16, 11, 12}, {2, 16, 21, 10}}}};
-
-const std::vector<InputShape> staticInputShapes4DC32 = {InputShape{// dynamic
-                                                                   {-1, 32, -1, -1},
-                                                                   // target
-                                                                   {{4, 32, 16, 14}, {16, 32, 5, 16}, {4, 32, 16, 14}}}};
-
-const std::vector<InputShape> dynamicInputShapes4D = {
-    InputShape{// dynamic
-               {ov::Dimension(1, 20), ov::Dimension(10, 40), ov::Dimension(10, 40), ov::Dimension(10, 40)},
-               // target
-               {{1, 32, 21, 10}, {2, 25, 11, 12}, {4, 15, 16, 14}, {7, 10, 20, 16}, {1, 32, 21, 10}}},
-    InputShape{// dynamic
-               {-1, -1, -1, -1},
-               // target
-               {{1, 24, 21, 8}, {2, 16, 11, 6}, {1, 24, 21, 8}}}
-};
-
-const std::vector<std::vector<size_t>> inputOrder4D = {
-        std::vector<size_t>{0, 1, 2, 3},
-        std::vector<size_t>{0, 2, 3, 1},
-        std::vector<size_t>{0, 2, 1, 3},
-        std::vector<size_t>{1, 0, 2, 3},
-        std::vector<size_t>{},
-};
-
-const std::vector<std::vector<size_t>> inputOrderPerChannels4D = {
-        std::vector<size_t>{0, 1, 2, 3},
-        std::vector<size_t>{0, 2, 1, 3},
-        std::vector<size_t>{1, 0, 2, 3},
-        std::vector<size_t>{},
 };
 
 const std::vector<CPUSpecificParams> CPUParams4D = {
         cpuParams_nChw16c,
         cpuParams_nChw8c,
-        cpuParams_nchw,
 };
 
 INSTANTIATE_TEST_SUITE_P(smoke_staticShapes4DC16_Transpose, TransposeLayerCPUTest,
```

Comment:
Could you please update the names of the instances as well.
smoke_staticShapes4DC16_TransposeBlocked

Reply:
fixed